### ISSUE TITLE
Experiment with using aria-valuetext for all alerts in this sim

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#318]
From #310, JAWS performance in FF with this sim (and others) is much worse than other platforms. Specifically, this causes alerts triggered after aria-valuetext updates to be lost by JAWS and never provided to the user.

One way around this could be to package the alert content into the aria-valuetext, because JAWS always reads the aria-valuetext. This would involve
- Modifying the aria-valuetext content pattern to include the alert when necessary
- Making sure the alert content is removed from the aria-valuetext before focus next lands on the arm/leg.
- Test to make sure this doesn't degrade the experience on other AT.

This work would not improve performance, but would probably make it so that JAWS in FF always provides the charge information to the user. Before working on this, we should see if @terracoda has any thoughts or concerns about this approach.

**Comment by terracoda** [comment]
OK, if I understand this correctly, we are trying to get better experience of hearing the number of electrons on the body. This has been a problem (not exactly the same) for Safari & Voice Over, too, but recent tweaks to the timing of the alert using the utterance queue addressed the issue of single electron alerts piling up and kind of getting in the way.

In the case for JAWS & FF we are trying to hear the electron count which is currently very rarely uttered. at all.

So, @emily-phet mentioned the plan was to "move electron count alerts into aria-value text". I think we used this approach to provide an improved sense of progress and directional changes for the Hand position slider, and it could possibly work for the Leg swing slider.

Currently for the leg, the aria-value text I think is quite simple, with two state-pattern somewhat like this:
1. Position {{position-number}}: foot {{rubbing on rug}}. or
2. Position {{position-number}}: foot {{off rug}}.

I think pedagogically we might only want to provide the electron count when the foot is actually "rubbing on rug" to make the connection that rubbing is required to cause a charge transfer, and then you's want to provide the latest total rather than individual counts, so the aria-valuetext might have to wait until the drag is finished? Does that make sense?

**Rough examples:**
Position 1, rubbing on rug, electrons on body 3.
Position 2, rubbing on rug, electrons on body 6.
Position 3, rubbing on rug, electrons on body 9.
Position 4, foot off rug.
Position 6, foot off rug.

Is this what you were imagining @emily-phet?

**Comment by emily-phet** [comment]
@jessegreenberg @terracoda 
After the decision yesterday to switch to Jaws + Chrome, this issue can be considered 'do not fix'. 
As we go forward, we should explore - and set some guidelines - for what content is provided as alert, and what is provided as aria-valuetext. 
Just wanted to let you both know that part of this issue isn't lost!

**Comment by jessegreenberg** [comment]
I believe this issue can be closed, @emily-phet can I make an issue in a11y-research to track exploring what should be provided in alerts vs aria-valuetext?

**Comment by jessegreenberg** [comment]
I made an issue above for that portion which is beyond performance and john-travoltage. Closing this one.
